include/grpc++/impl/codegen/status_code_enum.h

```diff
@@ -143,6 +143,11 @@ enum StatusCode {
   /// Unrecoverable data loss or corruption.
   DATA_LOSS = 15,
 
+  // Service was improperly implemented, violated a gRPC API requirement
```

Comment:
This error space is not owned by us, and would need to go through some... process to be added (will chat offline)